docs/en_us/internal/testing.rst

```diff
@@ -132,44 +132,50 @@ We use `nose <https://nose.readthedocs.org/en/latest/>`__ through the
 `django-nose plugin <https://pypi.python.org/pypi/django-nose>`__ to run
 the test suite.
 
-You can run all the python tests using ``paver`` commands. For example,
+For example, this command runs all the python test scripts.
 
 ::
 
     paver test_python
 
-runs all the tests. It also runs ``collectstatic``, which prepares the
+It also runs ``collectstatic``, which prepares the
 static files used by the site (for example, compiling Coffeescript to
 Javascript).
```

Comment:
nit totally outside the scope of what you offered to do: in both CoffeeScript and JavaScript, the "S" is capitalized